whois-api/src/main/java/net/ripe/db/whois/api/fulltextsearch/FulltextSearch.java

```diff
@@ -3,9 +3,12 @@
 import com.google.common.collect.Lists;
 import net.ripe.db.whois.api.rest.domain.Version;
 import net.ripe.db.whois.common.ApplicationVersion;
+import net.ripe.db.whois.common.rpsl.AttributeType;
 
 import java.io.IOException;
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public abstract class FulltextSearch {
 
```

Comment:
If these functions are being used only in one implementation , any reason ts in abstract class?

Reply:
Didn't realise that this was an abstract class used by Lucene implementation as well. I will move the method. Thank you!